src/compiler/program.ts

```diff
@@ -2232,7 +2232,10 @@ namespace ts {
                     if (isRedirect) {
                         inputName = getProjectReferenceRedirect(fileName) || fileName;
                     }
-                    if (getNormalizedAbsolutePath(checkedName, currentDirectory) !== getNormalizedAbsolutePath(inputName, currentDirectory)) {
+                    // Check if it differs only in drive letters its ok to ignore that error:

```

Comment:
Is it??

`C:\file` is indeed the same as `c:\file` — but it is most definitely NOT the same as `D:\file`.

Looking at the use below the code will wrongly assume `C:\file` match `D:\file`.

Reply:
That's guaranteed to be match because the file was found by its name.